### ISSUE TITLE
[ERROR] dashboard#index (I18n::MissingInterpolationArgument) "missing interpolation argument \"public...

### ISSUE THREAD

**Issue by lizconlan** [mysociety/belgium-theme#97]
```
An I18n::MissingInterpolationArgument occurred in dashboard#index:

  missing interpolation argument "public_body_name " in {:public_body_name=>"<a href=\"/nl_BE/body/politiezone_vesdre\">PolitieZone Vesdre (Verviers-Dison-Pepinster) n° 5289</a>", :info_request_title=>"<a href=\"/nl_BE/request/reglement_dordre_interieur_du_co\">Règlement d&#39;Ordre Intérieur du Conseil de Police de la Zone Vesdre</a>"} ("U zond een opvolgingsbericht naar {{public_body_name }} over \"{{info_request_title}}\"." given)
  lib/i18n_fixes.rb:35:in `block in gettext_interpolate'


-------------------------------
Request:
-------------------------------

  * URL        : https://transparencia.be/nl_BE/alaveteli_pro
  * HTTP Method: GET
  * IP address : [REDACTED]
  * Parameters : {"controller"=>"alaveteli_pro/dashboard", "action"=>"index", "locale"=>"nl_BE"}
  * Timestamp  : 2018-10-16 19:41:40 +0200

-------------------------------
Backtrace:
-------------------------------

  lib/i18n_fixes.rb:35:in `block in gettext_interpolate'
  lib/i18n_fixes.rb:31:in `gsub'
  lib/i18n_fixes.rb:31:in `gettext_interpolate'
  lib/i18n_fixes.rb:11:in `_'
  app/helpers/alaveteli_pro/dashboard_helper.rb:5:in `activity_item_description'
  app/views/alaveteli_pro/dashboard/_activity_list_item.html.erb:11:in `_app_views_alaveteli_pro_dashboard__activity_list_item_html_erb___1891720357684396980_70092367867940'
  app/views/alaveteli_pro/dashboard/index.html.erb:31:in `_app_views_alaveteli_pro_dashboard_index_html_erb__811410095504680284_70092341022420'
  app/controllers/application_controller.rb:115:in `record_memory'
  lib/strip_empty_sessions.rb:13:in `call'
```

**Comment by lizconlan** [comment]
This turns out to be a typo (that I missed, sorry!) that appears in a couple of places in Transifex (I fixed them in the editor so they should update correctly with the next translation hotfix for the current version when we do the 0.32 update - tagging in #77 to remind me to check carefully).

**tl;dr** - Spaces inside `{{ }}` markup for translated strings are significant and we had `{{public_body_name }}` instead of `{{public_body_name}}` for 2 variations of the followup message. This will break the dashboard view if the user has an `ActivityList::Item` of either affected type.

...but tracking it down was a bit tricky because...

It's from a constructed string that's built via

https://github.com/mysociety/alaveteli/blob/hotfix/0.31.0.4/app/helpers/alaveteli_pro/dashboard_helper.rb#L4-L6

Which meant having to double-check which model in https://github.com/mysociety/alaveteli/tree/hotfix/0.31.0.4/app/models/alaveteli_pro had triggered the error with its translated `#description` method.

The lazy check was that luckily 'opvolgingsbericht'  (follow-up) isn't used in too many places so a search in Transifex for `translation_text:opvolgingsbericht` then looking to find a matching pattern turned out to be useful \o/

**Comment by lizconlan** [comment]
Fixed via https://github.com/mysociety/alaveteli/tree/hotfix/0.31.0.5
